include/api.h

```diff
@@ -39,10 +39,10 @@
 #include <unistd.h>
 #endif /* __cplusplus */
 
-#define PROS_VERSION_MAJOR 4
-#define PROS_VERSION_MINOR 0
+#define PROS_VERSION_MAJOR 3
+#define PROS_VERSION_MINOR 7
 #define PROS_VERSION_PATCH 2
-#define PROS_VERSION_STRING "4.0.2"
+#define PROS_VERSION_STRING "3.7.2-dirty.243.f6c0e7d"
```

Comment:
```suggestion
#define PROS_VERSION_STRING "3.7.2-dirty.243.f6c0e7d"
```

Needs to be reverted